build-system/app-test.js

```diff
@@ -34,6 +34,28 @@ function appTestEndpoints(app) {
       ],
     });
   });
+
+  app.use('/date-picker/config.json', (req, res) => {
```

Comment:
I think @cathyxz split some non test app stuff out into separate files, which I think is a pattern we should follow so that this file doesn't end up being huge as well.

Reply:
I'm not sure which non test app stuff you're referring to, since that's what I created this file for, but I'd be glad to do something cleaner in a followup PR.

Reply:
Like this:

https://github.com/ampproject/amphtml/blob/98000ef8c85e92def6354a5b40c15ce41315f60b/build-system/app.js#L51

Basically, move the route logic into a separate file, then just import it from `app-test`, to keep this file from growing.

Reply:
Nice that looks pretty good. I will definitely do that in a followup PR